src/Files.App/ViewModels/UserControls/NavigationToolbarViewModel.cs

```diff
@@ -198,17 +206,56 @@ public bool IsSearchBoxVisible
 			}
 		}
 
-		private string? _PathText;
-		public string? PathText
+		private string? _LegacySharedPathPaletteText;
```

Comment:
Unless the property name is being reused, I would feel more comfortable if we kept the existing property names. Otherwise it will require additonal testing and a more thorough review.

Reply:
Thanks